src/main/resources/fileTemplates/j2ee/bukkit/Bukkit Submodule build.gradle.ft

```diff
@@ -1,4 +1,6 @@
-apply plugin: 'com.github.johnrengelman.shadow'
```

Comment:
I'm not sure if this is going to break in certain environments (*coughs* Forge), but if not - the `classpath` entry should be removed in the base `build.gradle`.

To be really explicit, I'm talking about the new `plugins` block not being usable by modules in Gradle versions past - if FG3 cannot be used by a Gradle version new enough that allows this, this file needs to be reverted.